src/main/java/com/onlinebookstore/store/repository/BookRepositoryImpl.java

```diff
@@ -0,0 +1,44 @@
+package com.onlinebookstore.store.repository;
+
+import com.onlinebookstore.store.model.Book;
+import com.onlinebookstore.store.util.HibernateUtil;
+import java.util.List;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+
+public class BookRepositoryImpl implements BookRepository {
+    private final SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
```

Comment:
Don't need to use HibernateUtil
```suggestion
    private final SessionFactory sessionFactory;
```